php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1121.html

```diff
@@ -22,10 +22,12 @@ <h2>Exceptions</h2>
 while (($line = next_line()) != NULL) {...}
 
 while ($line = next_line()) {...}
-{code}
 </pre>
 <h2>See</h2>
 <ul>
+  <li> MISRA C:2004, 13.1 - Assignment operators shall not be used in expressions that yield a Boolean value </li>
```

Comment:
for this one, it was actually doing well for removing the MISRA tags, but I wouldn't modify again the PR - we should avoid manually changing these files, and we'll update the rules again when the rule-api will be fixed.